From c55e9cb4d0fd5ebbbc2389b85c12e2f21fc775dd Mon Sep 17 00:00:00 2001
From: Theodor Gherzan <theodor@resin.io>
Date: Wed, 18 May 2016 14:43:57 +0200
Subject: [PATCH] am33x_evm.h: Change default dtb for the Beaglebone Green Wireless

Signed-off-by: Theodor Gherzan <theodor@resin.io>

Upstream-Status: Configuration

Index: git/include/configs/am335x_evm.h
===================================================================
--- git.orig/include/configs/am335x_evm.h
+++ git/include/configs/am335x_evm.h
@@ -143,7 +143,7 @@
 		"if test $board_name = A335BONE; then " \
 			"setenv fdtfile am335x-bone.dtb; fi; " \
 		"if test $board_name = A335BNLT; then " \
-			"setenv fdtfile am335x-boneblack.dtb; fi; " \
+			"setenv fdtfile am335x-bonegreen-wireless.dtb; fi; " \
 		"if test $board_name = BBG1; then " \
 			"setenv fdtfile am335x-bonegreen.dtb; fi; " \
 		"if test $board_name = A33515BB; then " \
