From 33dd194ba5ea9fe9d01faf6b08ac0f8b6fb98b9f Mon Sep 17 00:00:00 2001
From: Alexandru Costache <alexandru@balena.io>
Date: Tue, 15 Sep 2020 17:18:15 +0200
Subject: [PATCH] Install overlays in a different folder

Upstream-Status: Inappropriate [configuration]
Signed-off-by: Sebastian Panceac <sebastian@balena.io>
---
 Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makefile b/Makefile
index 0493664..84018c4 100644
--- a/Makefile
+++ b/Makefile
@@ -178,8 +178,8 @@ all_arch: $(ARCH_DTB)
 
 PHONY += install_arch
 install_arch: $(ARCH_DTBO)
-	mkdir -p $(DESTDIR)/lib/firmware/
-	cp -v $(obj)/*.dtbo $(DESTDIR)/lib/firmware/
+	mkdir -p $(DESTDIR)/boot/overlays
+	cp -v $(obj)/*.dtbo $(DESTDIR)/boot/overlays
 	mkdir -p $(DESTDIR)/usr/bin/
 	cp -v config-pin $(DESTDIR)/usr/bin/
 
-- 
2.7.4

